src/fmu/dataio/_definitions.py

```diff
@@ -61,6 +61,12 @@ def __post_init__(self):
     "timeseries": None,
 }
 
+STANDARD_TABLE_COLUMNS = {
```

Comment:
It refers to INDEX columns only, so perhaps better with `STANDARD_TABLES_INDEX_COLUMNS`?

Reply:
Agree, just note the slight modification, changed to STANDARD_TABLE_INDEX_COLUMNS